exercises/fulfill_a_promise/problem.md

```diff
@@ -16,43 +16,65 @@ the syntax is.  We are about to dive into that in detail.
 
 ## Setup
 
-To do many of the lessons in this workshop, you will need to install
-the `q` module.  To do this, type the following in your project's
-directory:
+To do many of the lessons in this workshop, you will need an installation of
+Node.js that supports ECMAScript 6 promises. That includes Node.js 0.12 or
+higher, and all versions of io.js.
+
+To find out if your Node.js installation has native promise support, execute
+the following in a Node.js REPL:
+
+```js
+typeof Promise !== "undefined"
+```
+
+If the REPL returns "true", then you are good to go! If not, you’ll have to use
+one of the many shims people have implemented. For this tutorial, we recommend
+`es6-promise` which aims to be strictly compliant to ES6 without any extra
+features. To use `es6-promise`, execute the following in a shell, preferably in the directory you’ll write your programs in:
 
 ```sh
-$ npm install q --save
+npm install es6-promise
 ```
 
-This will install a local copy of the `q` module into your directory
-which you can import into the file you have written using:
+Then, at the beginning of every file in this tutorial, add:
 
 ```js
-var q = require('q');
+var Promise = require('es6-promise').Promise
 ```
 
+Now you can use ES6 promises!
+
 ## Task
 
-Use the popular `q` library to create a promise.
+Create a promise using ES6 promise.
 
 Pass `console.log` to the `then` method of your promise.
 
-Manually resolve that promise using `setTimeout` with a delay of 300ms
+Manually fulfill that promise using `setTimeout` with a delay of 300ms
 and pass it a parameter of `"RESOLVED!"`.
```

Comment:
If we're switching to use the "fulfill" terminology (which is fine), then it's probably best if this is changed to `"FULFILLED"`.

Also this whole section (`Task`) is written with the kind of sequence naturally lending itself to Q, and not native promises (the word "manually" here is one example). Maybe it can be rewritten as something like: 

> Create a promise, using the `es6-promise` shim.
>
> Have it fulfill with the value `"FULFILLED!"` after a delay of 300ms, using `setTimeout`.
>
> Pass `console.log` to the `then` method of your promise.